src/components/AlertCard/styles.module.css

```diff
@@ -28,40 +32,60 @@
   margin-right: 0.5rem;
 }
 
+.flexGrow {
+  flex-grow: 2;
+  background-color: white;
+}
 .alertColumn > p,
 .subtitleText {
   margin-bottom: 0;
 }
 
 .spacerColumn {
   flex-grow: 2;
-  height: 65px;
-  background-color: #f0f0f0;
+  height: 4.0625rem;
 }
 
 .rightColumn {
   flex-grow: 2;
-  height: 65px;
+  height: 4.0625rem;
   text-align: right;
-  top: -65px;
-  margin-left: 850px;
-  padding-left: 150px;
+  top: -4.0625rem;
+  margin-left: 53.125rem;
+  padding-left: 9.375rem;
 }
 
 .rectangle {
```

Comment:
I think this class is unused

Reply:
@kem247 reminder here to remove this unused class

Reply:
fixed